tests/src/test/scala/com/nvidia/spark/rapids/GpuPartitioningSuite.scala

```diff
@@ -96,7 +96,7 @@ class GpuPartitioningSuite extends FunSuite with Arm {
 
   test("GPU partition with compression") {
     val conf = new SparkConf()
-        .set(RapidsConf.SHUFFLE_COMPRESSION_CODEC.key, "copy")
+        .set(RapidsConf.SHUFFLE_COMPRESSION_CODEC.key, "lz4")
```

Comment:
do we need to update the docs for the new option?

Reply:
updated.